# Detect distance and angle range
range_min : 0.0
range_max : 40.0
angle_min : -180.0
angle_max : 180.0
height_min : -0.3
height_max : 0.3 #0.5
# Robot itself
robot_x_max : 0.4
robot_x_min : -0.4
robot_y_max : 0.4
robot_y_min : -0.4
robot_z_max : 0.3
robot_z_min : -0.9
# Map metadata
robot_frame : "base_link"
map_resolution : 0.05
obs_size : 12  # 3 with shift
dilating_size : 15  # 5 with shift
map_size : 200
